analytics/transform_test.go

```diff
@@ -222,17 +220,30 @@ func BenchmarkToJson(b *testing.B) {
 	}
 }
 
-func TestToJsonWithGeo(t *testing.T) {
-	assert := assert.New(t)
-
-	// correct value
-	jsonEvent, err := stdJson.Marshal(eventMapWithGeo)
+// orderJson uses the standard json library to remarshal a []byte, resulting in a sorted JSON
+func orderJson(bytes []byte) ([]byte, error) {
```

Comment:
Checking my assumption here:

- We do this just to order the data so that we can have it in order in the test value (purely to make it more human-readable, no other reason)
- We only ever do this to the output of the function we're testing, not the test value

Are these two statements correct? 

If so, I'm ok leaving this in, there's something a bit weird about unmarshalling and remarshalling in a test but if it only happens to the value that's already been marshalled I think it's ok.

Reply:
Yes, this is only for the purpose of testing. It is not used to make it more human-readable, it is used instead to be able to assert the equality of the entire Jsons, as they are not ordered anymore now that we use jsoniter without key sorting.